File: semester 1/CppProgrammingandPractice/9week/OverloadingCompareOperator/Student.cpp
#include "Student.h"

std::ostream& operator<<(std::ostream& os, const Student& student) {
  os << "Id: " << student.id << " Name: " << student.name
     << " Grade: " << student.grade;
  return os;
}

int compareByGrade(const Student& a, const Student& b) {
  return a.grade > b.grade;
}

int Student::operator<(const Student& rhs) {
  return name.compare(rhs.name) < 0;
}

File: semester 2/PlatformBasedProgramming/GraduateStudentClass/src/Student.java
public class Student {


    private String id;
    private String name;
    private String email;

    public Student(String id, String name, String email) {
        this.id = id;
        this.name = name;
        this.email = email;
    }

    @Override
    public String toString() {
        return "Student ID: " + id + ", Name: " + name + ", Email: " + email;
    }


}